### ISSUE TITLE
Parse Json files in prince

### ISSUE THREAD

**Issue by gabri94** [AdvancedNetworkingSystems/prince#13]
A Json parser in prince must be implemented using json_c and a different interface must be defined to communicate over the graph_parser interface.

**Comment by gabri94** [comment]
```
struct topology{
    int id_lenght;                   //lenght of the id field (16B for ascii coded ipv4, 79 for ipv6)
    char *protocol;                //name of the protocol (olsr, oonf)
    char *self_id                    //id of the node 
    struct node *first;             //list of nodes

}

struct node{
    char *id;                                             //node identificator
    struct neighbor *neighbor_list;        //list of neighbors
}


struct neighbor{
    struct node *id;                     //pointer to neighbor node
    float weight;                          //cost of the link
    struct neighbor *next;          //next neighbor of the node
}
```

`

**Comment by gabri94** [comment]
implemented in 3cc0feffe470319ebc457a91e43491eb100d6157
